src/components/Signup/Signup.js

```diff
@@ -7,8 +7,8 @@ import {
   EmailFieldItem,
   FieldGroup,
   FormWithCaptcha,
-  InputFieldItem,
   CheckboxFieldItem,
+  InputFieldItem,
```

Comment:
Why did you moved it?

Reply:
😴 💤 💤

Reply:
because create other and when put again move it, i change